### ISSUE TITLE
Merge patches from Celane

### ISSUE THREAD

**Issue by tjyang** [pnp4nagios/pnp4nagios#13]
Hello @celane 
Glad you have you onboard. Write access is opened for you. 
Would you please merge your patches here ?

**Comment by sni** [comment]
do we want to have a quick look at the available branches first and sort out which one is most advanced and start from there?

**Comment by tjyang** [comment]
@sni 
Certainly. Thanks for the reminding.   As shown below. IMHO we should only keep master branch.
What do you think ? 

![image](https://user-images.githubusercontent.com/378638/235624638-c565b2de-d104-4dd3-8f17-fabd79c19c43.png)

**Comment by sni** [comment]
sure, the integration branch and such can be removed. But i was more talking about the different alternative forked branches which all tried to achieve the same (newer php support) in slightly different ways.

**Comment by tjyang** [comment]
Sorry, I got your point now.  I forgot you mentioned you have patches also. 

@celane was the [first comprehensive patches(](https://github.com/celane/pnp4nagios/commits/master)to my knowledge) cover all area of this stale project. I tested  it on http://nagios.pnp4nagios.org/nagios  and  instances in my test lab so I was biased initially. 

@celane, can you submit PR one-by-one from your personal repo and get @sni blessing/agreement ?
?

**Comment by tjyang** [comment]
@sni 
@celane 
@PVasileff 

I need to rename this repo as pnp4nagios.ToBeDeleted, and start a new fork and detach process.
This way we can work from @lingej 's last commit.

**Comment by tjyang** [comment]
https://github.com/pnp4nagios/pnp4nagios is now detached as standalone repo by github support. We start to commit carefully.
